dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -409,7 +413,13 @@ public Page<ItemRest> findByHandle(@Parameter(value = "handle", required = true)
                     "`dc.identifier.uri` wasn't found.");
         }
 
-        List<Item> itemList = clarinItemService.findByHandle(context, metadataField, handle);
+        String handlePrefix = configurationService.getProperty("handle.canonical.prefix");
```

Comment:
why?

Reply:
added docs

Reply:
what about different prefixes?

Reply:
I removed that code because it was a small addition I used once during development, and it is no longer needed. I considered keeping it, but since you asked about prefixes and the fix wasn't straightforward, I decided to remove the unused code. The endpoint still works properly, but you always need to search by the full handle (fixed prefix).